modules/delegates.js

```diff
@@ -272,25 +281,30 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
+	var secretsList;
 
 	if (library.config.forging.secret) {
 		if (Array.isArray(library.config.forging.secret)) {
```

Comment:
`library.config.forging.secret` should always be an array because it's required by schema